docs/sources/operators-guide/deploy-grafana-mimir/jsonnet/deploying.md

```diff
@@ -100,3 +100,5 @@ You can use [Tanka](https://tanka.dev/) and [jsonnet-bundler](https://github.com
      ```
 
    > **Note**: The generated Kubernetes manifests create resources in the `default` namespace. To use a different namespace, change the `namespace` configuration option in the `environments/default/main.jsonnet` file, and re-generate the Kubernetes manifests.
+
+   > **Note**: Jsonnet-based installation uses etcd for the HA tracker by default. To install etcd, we expect you have the [etcd-operator](https://github.com/coreos/etcd-operator) running in your K8S cluster.
```

Comment:
```suggestion
   > **Note:** A Jsonnet-based installation uses etcd for the HA tracker by default. Before installing etcd, make sure that you have have the [etcd-operator](https://github.com/coreos/etcd-operator) running in your K8S cluster.
```

Reply:
@osg-grafana I would clarify that etcd cluster is created by jsonnet. What's about the following?

```suggestion
   > **Note:** A Jsonnet-based installation uses etcd for the HA tracker by default. The jsonnet-based installation creates the etcd using the [etcd-operator](https://github.com/coreos/etcd-operator). Before applying the `kubectl apply` command, make sure that you have the etcd-operator running in your Kubernetes cluster.
```